department-of-veterans-affairs/va.gov-cms
# VACMS-1902: Sort drupal spec tool behat output intelligently.

## Description

See #1902. 

## Testing done

None. This just sorts the code. 

## Screenshots


## QA steps

Visit the [Bundles](https://docs.google.com/spreadsheets/d/1vL8rqLqcEVfESnJJK_GWQ7nf3BPe4SSevYYblisBTOI/edit#gid=858986005) and [Fields](https://docs.google.com/spreadsheets/d/1vL8rqLqcEVfESnJJK_GWQ7nf3BPe4SSevYYblisBTOI/edit#gid=943298572) tabs and verify that they are sorted by 

1. Entity type
2. Entity bundle
3. Field machine name (for the fields tab)
